Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/medic.yml

```
@@ -211,7 +211,8 @@
       - id: CMVendorBundleSquadMedicalApparel
       - id: CMMRE
```

Small nitpick but in 13 the gloves are above the MRE
```suggestion
      - id: CMVendorBundleSquadMedicalApparel
      - id: RMCHandsLatexMarine
        name: combat sterile gloves
      - id: CMMRE
```